sklearn/feature_extraction/tests/test_text.py

```diff
@@ -892,6 +892,17 @@ def test_countvectorizer_vocab_dicts_when_pickling():
         assert_equal(cv.get_feature_names(), unpickled_cv.get_feature_names())
 
 
+def test_countvectorizer_string_object_as_input():
+    # Ensure that string object is not given as input.
+    cv = CountVectorizer()
+    exception = ValueError
+    test_str = "hello world!"
+
+    assert_raises(exception, cv.fit_transform, test_str)
```

Comment:
oops, it was `assert_raise_message` before and I suggested a change to `assert_raises`. [Been looking too much at the tree tests...](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tree/tests/test_tree.py#L496)